### PR TITLE
Add libzip project

### PR DESCRIPTION
**Pull request by hsccr**

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -834,6 +834,24 @@ def build(self):
...
+    def __init__(self):
+        Project.__init__(self,
+            'libzip',
+            archive_url = 'http://nih.at/libzip/libzip-1.1.3.tar.gz',
+            dependencies = ['zlib'],
+            )
+
+    def build(self):
+        cmake_config = 'Debug' if self.builder.opts.configuration == 'debug' else 'RelWithDebInfo'
+        self.exec_vs(r'cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX="%(gtk_dir)s" -DGTK_DIR="%(pkg_dir)s" -DCMAKE_BUILD_TYPE=' + cmake_config,add_path=self.builder.opts.cmake_path)
```
Comment (nacho) [inline build.py]: you are missing one space after ,

Comment (hsccr) [inline build.py]: I can not find missing.
please explain in more detail.

**Comment by nacho** [comment]
I just mean change "cmake_config,add_path=self.builder.opts.cmake_path)" for "cmake_config, add_path=self.builder.opts.cmake_path)"

**Comment by nacho** [comment]
Looks good now. Please squash both commits into one single commit and I will merge it.

**Comment by nacho** [comment]
Thanks for the patch!

**Pull request merged**